modules/apps/commerce/commerce-api/src/main/java/com/liferay/commerce/order/importer/type/CommerceOrderImporterType.java

```diff
@@ -37,10 +38,18 @@ public Object getCommerceOrderImporterItem(
 
 	public String getCommerceOrderImporterItemParamName();
 
+	public List<CommerceOrderImporterItem> getCommerceOrderImporterItems(
+			CommerceOrder commerceOrder, FDSPagination fdsPagination,
+			Object object)
+		throws Exception;
+
 	public List<CommerceOrderImporterItem> getCommerceOrderImporterItems(
```

Comment:
Do we need to keep the "old" method here?

Reply:
its not necessary, I can remove it and update the areas its being used with the newer one.